chips/lowrisc/src/aon_timer.rs

```diff
@@ -142,35 +142,6 @@ impl AonTimer {
         ms.saturating_mul(self.aon_clk_freq).saturating_div(1000)
     }
 
-    /// Start the wake up counter
-    #[cfg(aon_wkup_timer)]
-    fn wkup_start_count(&self) {
-        self.registers.wkup_ctrl.write(WKUP_CTRL::ENABLE::SET);
-    }
```

Comment:
Should probably re-add these features for now.

Seeing this code makes me skittish generally, but this is not for this PR to fix. These differences in behavior should really be reflected on a type-level, instead of compile-time features.

Reply:
This is dead code though, it's unused and I assume untested.

I think if we want to add a feature to enable/disable this it should be it's own PR where someone tests that it works and then re-adds it, with justification of why it's behind a feature flag

Reply:
We don't know it's dead code. A board outside of tree might be enabling this config.

Reply:
We do know that no one is enabling the config though, as there is currently no config to enable